packages/discord.js/src/managers/GuildForumThreadManager.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const { Routes } = require('discord-api-types/v10');
+const ThreadManager = require('./ThreadManager');
+const { TypeError, ErrorCodes } = require('../errors');
+const MessagePayload = require('../structures/MessagePayload');
+const { resolveAutoArchiveMaxLimit } = require('../util/Util');
+
+class GuildForumThreadManager extends ThreadManager {
+  /**
+   * Options for creating a thread. <warn>Only one of `startMessage` or `type` can be defined.</warn>
+   * @typedef {StartThreadOptions} GuildForumThreadCreateOptions
+   * @property {MessageOptions|MessagePayload} message The message associated with the thread post
```

Comment:
`MessageOptions` doesn't exist. In any case, that should be replaced with a new `@typedef` called `GuildForumThreadMessageCreateOptions`?

Reply:
This doesn't appear to be resolved?